cypress/e2e/pages/governance/create/index.spec.ts

```diff
@@ -0,0 +1,593 @@
+const viewPorts = ["macbook-16", "ipad-2", "iphone-x"];
+
+viewPorts.forEach((viewPort) => {
+  context(`/governance/create on ${viewPort}`, () => {
+    const nameOfProposal = "Test Proposal";
+    const discussion = "https://github.com/DeFiCh/dfips/issues/243";
```

Comment:
can we also add validation of reddit discussion, that is doesn't throw an error in input, thanks!

Reply:
added test cases to check for reddit discussion !